dashboard/test/models/concerns/user/age_test.rb

```diff
@@ -0,0 +1,147 @@
+require 'test_helper'
+
+class AgeTest < ActiveSupport::TestCase
+  include Minitest::RSpecMocks
+  describe '#under_13?' do
+    let(:user) {create(:user, age: 10)}
+    context 'when the user is under 13' do
+      it 'returns true' do
+        _(user.under_13?).must_equal true
+      end
+    end
```

Comment:
nit: We are calling the same user method in all assertions, for consistency we could be using the subject blocks.
```suggestion
  describe '#under_13?' do
  subject(:under_13?) {user.under_13?}
    let(:user) {create(:user, age: 10)}
    context 'when the user is under 13' do
      it 'returns true' do
        _under_13?.must_equal true
      end
    end
```